konig-gcp-deployment-manager/src/main/java/io/konig/gcp/deployment/GoogleBigqueryTableVisitor.java

```diff
@@ -64,10 +64,10 @@ private void handleStorageBucket(Graph graph, GoogleCloudStorageBucket bucket) {
 		String bucketName = bucket.getName();
 		String configName = manager.storageBucketConfigName(bucketName);
 		
-		StorageBucketResource resource = manager.getConfig().findResource(configName, StorageBucketResource.class);
+		StorageBucketResource resource = manager.getConfig().findResource(bucketName, StorageBucketResource.class);
```

Comment:
Instead of using bucketName as an argument to findResource, you should change the implementation of manager.storageBucketConfigName so that it returns bucketName (without the "bucket-" prefix).
This way we have a consistent pattern of getting the config name from the manager regardless of resource type.